docs/guides/building/linux.md

```diff
@@ -63,6 +63,11 @@ If your app plays audio/video you need to enable `tauri.conf.json > tauri > bund
 
 :::
 
+## Snap Package
+
+Snap packages are an alternative distribution format that also does not rely on the system installed packages. Snap was created by Canonical. Snap packages work across most linux distributions. In order to build snap packages using the CLI, the `snapcraft` package must be installed. On the other hand, users are required to have the `snapd` package installed on their system. After building your snap package, it can be signed and uploaded to the official Snap Store for easier distribution. The output file can be installed using (`sudo snap install ./MyProject.snap --dangerous`). The dangerous flag is required to install the package. In order to bypass the --dangerous flag and sign your package, a Snap Store account is required and you must login via. snapcraft on your local system using (`snapcraft login`).
```

Comment:
```suggestion
[Snap packages](https://snapcraft.io/about) are an alternative distribution format for Linux that do not rely on the system installed packages. In order to build snap packages using the CLI, the [`snapcraft`](https://github.com/snapcore/snapcraft) package must be installed on the developer's machine.

After building a snap package, it can be signed and uploaded to the [Snap Store](https://snapcraft.io) for easier distribution. More information about Snaps, how to use them, and usage for end users can be found on the [Snapcraft about page](https://snapcraft.io/about)
```

I put a few tweaks in here and pointed to where I felt was the relevant place for a developer to find more information (although I'm not at all an expert in this area). I prefer to link out to upstream docs and only cover any Tauri-specific configuration details here in the docs which I think this should accomplish.